### PR TITLE
[DEVOPS-859] release nix doc updates

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

File: nix-darwin/README.md [hunk 1]
```diff
@@ -21,27 +21,17 @@ host* (your laptop probably).
...
-   be buildkite or hydra.
+1. Build and run prepare script
 
-       nix-build -I iohk-ops=https://github.com/input-output-hk/iohk-ops/archive/develop.tar.gz '<iohk-ops/nix-darwin>'
-       ./result/bin/deploy [ buildkite | hydra ]
+       nix-build -A tools
+       result/bin/prepare-mac user@host [buildkite|hydra]
 
-   This will prepare the system so that `nix-darwin` can be installed.
+   This will prepare the system so that the mac can be deployed.
```
Comment (rvl) [inline nix-darwin/README.md]: Add a note that this will nuke any existing nix setup on the target mac.

**Review by rvl — APPROVED** [review]
Thanks

**Review by jbgi — APPROVED** [review]

**Pull request closed without merging**